### DISCUSSION TITLE
.png icon transparency issue for termux shortcut

### DISCUSSION THREAD

**Discussion by Android-X13** [termux/termux-app#2609] [Q&A] [answered]
I have placed a .png icon with transparent background for a particular shortcut inside the ~/.shortcuts/icons folder and it shows up in the home screen as expected on a Huawei device. However that same icon loses its transparency on a Samsung S10 device (it is still displayed on the home screen but instead of having a transparent background it has a white one).
Anyone knows why that happens? Is this issue related to Termux or the Samsung launcher?

**Comment by agnostic-apollo** [comment, marked as answer]
png files don't have "backgrounds". If Samsung launcher is showing it, then issue would likely be with that. You can also try other launchers like nova to test and confirm.

`termux-widget` does not make any modifications to image.

https://github.com/termux/termux-widget/blob/v0.13.0/app/src/main/java/com/termux/widget/TermuxCreateShortcutActivity.java#L142
